internal/security/setup/option/exitCode.go

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under
+// the License.
+//
+// SPDX-License-Identifier: Apache-2.0'
+//
+
+package option
+
+// ExitCode is the code used for exit status
+type exitCode int
```

Comment:
why have a new type for the exit code and not just define constants for the various exit codes we use for specific situations? seems like an unnecessarily abstraction

Reply:
It is less verbose in the constant name without losing the context- if only defined in constant, then one would end up with 
```
const(
   normalExitCode = 0
   noOptionSelectedExitCode = 1
   exitWithErrorExitCode = 2
)
```